tests/Custom/InstantiationTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Behat\Mink\Tests\Driver\Custom;
+
+use Behat\Mink\Driver\ZombieDriver;
+
+class InstantiationTest extends \PHPUnit_Framework_TestCase
+{
+    public function testInstantiateWithServer()
+    {
+        $server = $this->prophesize('\Behat\Mink\Driver\NodeJS\Server\ZombieServer')->reveal();
```

Comment:
As Prophecy is now included natively in PHPUnit since 4.5 (which is already quite old), I'm using it as I find its usage much easier than the PHPUnit mocks (and this is the reason why support is builtin in PHPUnit btw)